iroha/src/config.rs

```diff
@@ -91,6 +91,19 @@ impl Configuration {
         Ok(())
     }
 
+    /// Load trusted peers variables from a json *pretty* formatted file.
+    pub fn load_trusted_peers_from_path<P: AsRef<Path> + Debug>(
+        &mut self,
+        path: P,
+    ) -> Result<(), String> {
+        let file = File::open(path).map_err(|e| format!("Failed to open a file: {}", e))?;
+        let reader = BufReader::new(file);
+        let trusted_peers: BTreeSet<PeerId> = serde_json::from_reader(reader)
+            .map_err(|e| format!("Failed to deserialize json from reader: {}", e))?;
+        self.sumeragi_configuration.trusted_peers = trusted_peers;
+        Ok(())
+    }
+
```

Comment:
If we extracted trusted peers configuration from main configuration file, would be better to create an additional structure to load and store this new configuration.

Reply:
@humb1t
Is it means separate `trusted peers` from `sumeragi configuration`? or Make the structure of trusted peers and is used when only load config as follows?
```
let trusted_peers: TrustedPeers = serde_json::from_reader(reader)
    .map_err(|e| format!("Failed to deserialize json from reader: {}", e))?;
self.sumeragi_configuration.trusted_peers = trusted_peers;
```

Reply:
@MizukiSonoko I think second option is good enough.

Reply:
Or something like this, I think
```rust
let trusted_peers = TrustedPeers::from_path(&path)?;
self.sumeragi_configuration.trusted_peers = trusted_peers;
```

Reply:
```
self.sumeragi_configuration.trusted_peers = TrustedPeers::from_path(&path)?;
```
=)